cubic-server/interface/LogsInterface.cpp

```diff
@@ -1,9 +1,10 @@
 #include <string>
 #include <iostream>
 #include <fstream>
+#include <queue>
 #include <unistd.h>
 
-//#include "Logger.hpp"
+#include "../logging/Logger.hpp"
```

Comment:
Please leave a comment here if you know how to fix this include

Reply:
Shouldn't `#include "logging/Logger.hpp"` work alright ?

Reply:
I'll try and see if it works